docs/reference/nodes/conditional.md

```diff
@@ -0,0 +1,23 @@
+# Conditional Nodes
+
+## ConditionEquals Node
+>[!NOTE]
+> This node is currently a *Beta* operator. Beta operators are supposed to be used for testing purposes only and their behavior may change in the future.
+
+The *ConditionalEquals* node allows to compare if two values are equal. Depending on the result of the comparison, the execution of the workflow will continue through one graph branch or another. 

```

Comment:
Maybe here is needed a note to mention avoiding connecting plugs between nodes of the two branches? 
I mean, because for example, one node of one branch cannot be dependent of the output data from a node of the other branch.

Reply:
Can you attach an example workflow of what setup you refer to? I think I get it, but just to make it clear

Reply:
Hmm I understand that this may not be a desired logic in many cases, but it does not break anything, right? I mean, it is like using a variable in a code that gets changed by an if statement that you use later in the code. 
Do you think is worth adding a recommendation node about this anyways?

Reply:
Well, if it doesn't break anything we can leave it, when the users test that case they will realise that the other branch won't be executed. 👍